### ISSUE TITLE
Tests Ran / Passed / Failed Per Test Script on Command Line

### ISSUE THREAD

**Issue by knightofiam** [AlexDarigan/WAT#293]
It would be nice to go from this:

```
-------RESULTS-------
Took 1 second(s)
0 Test Scripts Crashed
9 / 10 Test Scripts Passed
-------RESULTS-------
```

to something like this:

```
-------RESULTS-------

Took 1 second(s)

Test Scripts:

0 Crashed
8 / 10 Passed
2 / 10 Failed

Tests:

221 / 225 Passed
3 / 225 Failed

Details:

1. FAILED TEST SCRIPT: MyTestScript5:
  1. FAILED TEST: MyTest23
  2. FAILED TEST: MyTest37

2. FAILED TEST SCRIPT: MyTestScript7:
  1. FAILED TEST: MyTest2

-------RESULTS-------
```

**Comment by AlexDarigan** [comment]
I'll get around to this eventually no doubt.

**Comment by AlexDarigan** [comment]
> I'll get around to this eventually no doubt.

Seems like I lied. There is enough information as is now to figure out the failures so I'm going to close this issue instead. Just need to prioritize other things.

**Comment by AlexDarigan** [comment]
Reopening unresolved issue.

**Comment by AlexDarigan** [comment]
![image](https://user-images.githubusercontent.com/47497501/234059389-d2a52e6d-4d80-4170-b923-044eb0d3d3ef.png)

Implemented in a17e05937af266637006ab791548ae1fe33ebad2

**Comment by knightofiam** [comment]
That’s great! Thank!
